### PR TITLE
Post : ES6 template strings

### PR DESCRIPTION
**Pull request by jbleuzen**
Needs some adjustements and review

### CODE REVIEW

File: content/fr/articles/js/es2015/template-strings.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+tags:
+  - JavaScript
+  - ES6
+  - ES2015
+authors:
+  - jbleuzen
+---
+
+ES2015 ajoute le support des template strings qui va permettre enfin de se 
+simplifier la vie lorsqu'il doit manipuler des chaines de caractères.
```
Comment (MoOx) [inline content/fr/articles/js/es2015/template-strings.md]: Etrange "lorsqu'il doit", il qui ?

**Comment by MoOx** [comment]
Except my little comment, this post looks good as it is.

**Comment by MoOx** [comment]
Note: You have a weird "merge" commit

**Comment by magsout** [comment]
:shipit:

File: content/fr/articles/js/es2015/template-strings.md (hunk 1 above)
Comment (jbleuzen) [inline content/fr/articles/js/es2015/template-strings.md]: -il +on

**Comment by MoOx** [comment]
I am merging since we are "late". Feel free to improve it.

**Pull request merged**